static/app/components/deprecatedSmartSearchBar/searchBarDatePicker.tsx

```diff
@@ -1,5 +1,6 @@
 import {useEffect, useRef, useState} from 'react';
 import styled from '@emotion/styled';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import format from 'date-fns/format';
```

Comment:
I disabled the linter here because I don’t want to cause regressions, however, it might also be possible that we can delete this `deprecatedSmartSearchBar` altogether because it’s only used in one place as a fallback in case users don’t yet have the [hasNewSpansIUFlag](https://github.com/getsentry/sentry/blob/master/static/app/views/performance/transactionSummary/transactionSpans/spanDetails/content.tsx#L56-L63) feature flag.

If this feature flag is generally available, a lot of code could go away.